content/cloud-docs/api-docs/teams.mdx

```diff
@@ -85,6 +85,7 @@ $ curl \
       "type": "teams",
       "attributes": {
         "name": "team-creation-test",
+        "sso-team-id": "cb265c8e41bddf3f9926b2cf3d190f0e1627daa4",
```

Comment:
This field is only returned if the organization that this team belongs to is in the Business tier. How should I express this?

Reply:
My instinct is to have one sentence under the Sample Response header that explains this.  Thoughts on this approach? I think it's better to state this in each place where the user could see this and potentially get confused. So each response that has this attribute in it would now have this brief explanatory line. I don't want to put it in a note because that feels like it would get visually very busy if we used it throughout the page. So something like...

### Sample Response

The `sso-team-id` attribute is only returned if this team's organization is in the Terraform Cloud Business tier.

Reply:
Updated this based on your comment @laurapacilio